lib/driver.js

```diff
@@ -333,14 +333,20 @@ class XCUITestDriver extends BaseDriver {
 
     await this.runReset();
 
+    let isLogCaptureStarted;
     const startLogCapture = async () => {
       const result = await this.startLogCapture();
       if (result) {
         this.logEvent('logCaptureStarted');
       }
       return result;
     };
-    const isLogCaptureStarted = await startLogCapture();
+
+    if (this.opts.skipLogCapture) {
+      log.info("'skipLogCapture' is set. Skipping starting logs such as crash, system, safari console and safari network.");
```

Comment:
This log line is repeated twice. Perhaps it would be better to move the check into the `startLogCapture` function on line 337?
```js
const startLogCapture = async () => {
  if (this.opts.skipLogCapture) {
    log.info("'skipLogCapture' is set. Skipping starting logs such as crash, system, safari console and safari network.");
    return false;
  }

  const result = await this.startLogCapture();
  if (result) {
    this.logEvent('logCaptureStarted');
  }
  return result;
};
```

Reply:
make sense than mine 👍